docs/deploying/deploy-nextjs-app.mdx

```diff
@@ -50,6 +50,29 @@ The configuration parameters are described below. Make sure to update the values
 
 Array of blockchain networks where your dapp is deployed. Use values from [`viem/chains`](https://viem.sh/docs/chains/introduction.html).
 
+<details>
+<summary>How to add a chain to the frontend using viem/chains</summary>
+
+To add a new chain to your frontend:
+
+1. Import the chain from `viem/chains` in your `scaffold.config.ts` file.
+2. Add the imported chain to the `targetNetworks` array.
+
+Here's an example of how to add the Optimism network:
+
+```ts
+import { hardhat, optimism } from "viem/chains";
+
+export const scaffoldConfig = {
+  targetNetworks: [hardhat, optimism],
+  // ... other config options
+};
+```
+
+</details>
```

Comment:
Umm maybe this seems too long and a bit unnecessary?  

"Array of blockchain networks where your dapp is deployed. Use values from [`viem/chains`](https://viem.sh/docs/chains/introduction.html) eg: `targetNetworks: [chains.optimism]`"

Reply:
Updated! I wanted to go for the absolute newbies when creating the PR but this seems good too